pkgs/development/python-modules/spectra/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPythonPackage, colormath, fetchPypi, lib }:
+
+buildPythonPackage rec {
+  version = "0.0.11";
+  pname = "spectra";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-jrNipRh8tjzuE80BGGeZwMeRo6077FeyeRMuElIXYrg=";
+  };
+  doCheck = false;
```

Comment:
Any chance to run the tests (at least partially), or to find out why it won't work?

Reply:
I've pushed a fix using checkTarget but it does not seem to run the tests ("0 tests ran"). Do you know why ?